src/clr-addons/quick-list/quick-list.component.scss

```diff
@@ -0,0 +1,11 @@
+.pl-0-75 {
+  padding-left: 0.75rem;
+}
+
+.pb-0-5 {
+  padding-bottom: 0.5rem;
+}
+
+.pt-0-5 {
```

Comment:
please don't name our classes in the naming scheme of bootstrap to avoid future collisions.